libs/platform/src/lib/table/table.component.spec.ts

```diff
@@ -1201,8 +1201,8 @@ class TreeTableDataProviderMock extends TableDataProvider<SourceTreeItem> {
                 calculateTableElementsMetaData();
 
```

Comment:
My only comment to this PR, would be add more unit tests and clearly validate before and after, first to last, last to first, last in between, first in between works correctly. I don't see any added tests here.